ajax/libs/utf8/2.1.1/utf8.js

```diff
@@ -0,0 +1,244 @@
+/*! https://mths.be/utf8js v2.0.0 by @mathias */
```

Comment:
Should say `v2.1.0` as in https://github.com/mathiasbynens/utf8.js/blob/v2.1.1/utf8.js. (This was incorrect before, but it’s fixed now.)